lightwood/encoders/time_series/rnn.py

```diff
@@ -149,26 +197,76 @@ def encode(self, column_data, get_next_count = None):
         else:
             return self._pytorch_wrapper(torch.stack(ret)), self._pytorch_wrapper(torch.stack(next))
 
+    def _decode_one(self, hidden):
+        """
+        Decodes a single time series from its encoded representation.
+        :param hidden: time series embedded representation tensor, with size self._encoded_vector_size
+        :return: decoded time series list
+        """
+        self._decoder.eval()
+        with torch.no_grad():
+            ret = []
+            next_tensor = torch.full((1, 1, self._n_dims), self._sos, dtype=torch.float32).to(self.device)
+            for _ in range(self._max_dec_length):
+                next_tensor, hidden = self._decoder.forward(next_tensor, hidden)
+                ret.append(next_tensor)
+            return torch.stack(ret)
```

Comment:
Not really a not on this PR specifically, but I only now realize this method exist and I think this is done manually in a few places by adding a dimension + `concat` , if you spot this anywhere else feel free to replace with `stack`, it makes much more sense.